samples/qa-a11y/src/views/SliderButton.js

```diff
@@ -0,0 +1,22 @@
+import SliderButton from '@enact/agate/SliderButton';
+import React from 'react';
+
+import Section from '../components/Section';
+
+import appCss from '../App/App.module.less';
+
+const SliderButtonView = () => (
```

Comment:
I got console.error when i select button.

Uncaught TypeError: onChange is not a function